django_project/utils/urls.py

```diff
@@ -1,8 +1,18 @@
 from django.urls import path
-
 from . import views
 
+"""
+	TODO: 
+	1. Add URL subpaths here.
+	2. Write functions in views.py to handle the various subpaths that you added.
+
+	Examples of other subpaths cound be:
+    - path('post/<post_id>', views.posts, name='posts')
+    - path('data/<date>/<time>', views.data, name='data')
```

Comment:
I tried double checking that these examples work but I wasn't able to get them to work. I might be doing something wrong. Here is what I did:
In views.py
```
def data(date,time):
	#Get soup object
	soup = getSoup()
	#Insert a meta tag
	insertMetaTag(soup, "og:title", date)
	#Return soup object
	return HttpResponse(soup.prettify())
```
In urls.py
```
urlpatterns = [
    path('data/<int:date>/<int:time>', views.data, name='data'),
    path('', views.basePage, name='basePage'),
    path('<subpath>/', views.withSubpath, name='withSubpath'),
]
```